src/SourceBuild/tarball/content/build.sh

```diff
@@ -82,13 +83,26 @@ while :; do
     shift
 done
 
+if [ "$CUSTOM_PACKAGES_DIR" != "" ]; then
+  if [ -d "$CUSTOM_PACKAGES_DIR" ]; then
```

Comment:
Is the `[ -d "$CUSTOM_PACKAGES_DIR" ]` check here necessary because it is already validated in the argument parsing logic.

Reply:
Good point, removed.